packages/editor/src/client/atoms/tourSteps.tsx

```diff
@@ -0,0 +1,96 @@
+import {Step} from 'react-shepherd'
+import {wait} from '../utility'
+
+const navButtons = {
+  exit: {
+    text: 'Exit',
```

Comment:
i mention this down bellow but all the strings that are displayed should be in the translation file.